### PR TITLE
[D] Add datacenter capability details.

### PR DESCRIPTION
**Pull request by michaelkad**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
=== RUN   TestAccIBMPIDatacentersDataSourceBasic
--- PASS: TestAccIBMPIDatacentersDataSourceBasic (10.88s)
PASS

=== RUN   TestAccIBMPIDatacenterDataSourceBasic
--- PASS: TestAccIBMPIDatacenterDataSourceBasic (7.28s)
PASS
```

### CODE REVIEW

File: ibm/service/power/data_source_ibm_pi_datacenter.go [hunk 1]
```diff
@@ -28,6 +29,124 @@ func DataSourceIBMPIDatacenter() *schema.Resource {
 			},
 
 			// Attributes
+			Attr_CapabilityDetails: {
```
Comment (ismirlia) [inline ibm/service/power/data_source_ibm_pi_datacenter.go:32]: That's really rough. Did you ever get a chance to ask esdras to see if they could cut out one of the middle layers?

**Comment by ismirlia** [comment]
Could you also add or update the datacenter terraform test to make sure those new fields are populated correctly? I don't know if you got a chance run the data source manually and see what that ends up looking like.

File: ibm/service/power/data_source_ibm_pi_datacenter.go (hunk 1 above)
Comment (michaelkad) [reply ibm/service/power/data_source_ibm_pi_datacenter.go:32]: That was updated.

**Review by ismirlia — APPROVED** [review]

**Review by Alexander-Kita — APPROVED** [review]

**Comment by ismirlia** [comment]
Opened https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5666, closing this one.

**Pull request closed without merging**